patterns.md

```diff
@@ -156,19 +156,30 @@ we pass a dictionary containing them all.
 
 ### Layered approach
 
-An application should be split into three layers, each with their own role:
+An application should be split into architectural layers, each with their own role, 
+each ignorant of lower-level layers.  
+
+Here are our current layers, listed in order with the lowest-level layer first.
 
 - `interfaces` - where functionality for websites, command-line applications,
-  Celery tasks live. This covers anywhere where some external event (ie a HTTP
+  Celery tasks live. This covers anywhere where some external event (eg a HTTP
   request) triggers a query or an action. No application logic should live in
-  this layer, just the translation of the interface event into a domain-layer call.
+  this layer, just the translation of the interface event into a application- or
+  domain-layer call.
+
+- `application` - where the "use-cases" of the application live. Use-cases are
```

Comment:
Don't need it for MR - will remove.

I find it useful to the others to indicate they are a single thing - makes it read better in my opinion. Otherwise, my brain starts reading the "use" or "move" as a verb (as in "Use cases to transport things").